x-pack/plugins/observability/server/lib/rules/threshold/register_threshold_rule_type.ts

```diff
@@ -132,7 +132,7 @@ export function thresholdRuleType(
           groupBy: schema.maybe(schema.oneOf([schema.string(), schema.arrayOf(schema.string())])),
           filterQuery: schema.maybe(
             schema.string({
-              validate: validateIsStringElasticsearchJSONFilter,
+              validate: validateKQLStringFilter,
```

Comment:
@kdelemme We should add this to our schemas for anywhere we accept KQL.

Reply:
Sure :)